src/main/java/org/alliancegenome/curation_api/model/entities/associations/variantAssociations/CuratedVariantGenomicLocationAssociation.java

```diff
@@ -35,5 +46,18 @@
 )
 
 public class CuratedVariantGenomicLocationAssociation extends VariantGenomicLocationAssociation {
-
+	
+	@IndexedEmbedded(
+		includePaths = {
+			"variantTranscript.name", "variantTranscript.modEntityId",
+			"variantTranscript.modInternalId", "variantTranscript.curie",
+			"vepConsequence.name", "variantTranscript.name_keyword", 
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChBlNjA0ZjJjMzY4MzBmMDE4EgpjaGVja3N0eWxl -->